### DISCUSSION TITLE
Invoke-VCFHealthReport requiring Tar Utility

### DISCUSSION THREAD

**Discussion by AdminAlley** [vmware/powershell-module-for-vmware-cloud-foundation-reporting#223] [Community Help] [answered]
I was able to run the Alert & Overview reports without any issues. When I ran the Invoke-VCFHealthReport cmdlet on a VCF 5.1 environment, I'm told I need the tar utility. Haven't been asked for this before. Is this new or a possible issue with the environment I'm running it against?

<img width="830" alt="image" src="https://github.com/vmware/powershell-module-for-vmware-cloud-foundation/assets/95247386/9b8f62f1-55f9-4605-98e2-0c7aa05b7aa6">

**Comment by tenthirtyam** [comment, marked as answer]
The module requires 2019 or later if using Windows Server which have tar.exe included. Check the Windows version.

What's your OS?

**Comment by tenthirtyam** [comment]
Refer to https://vmware.github.io/powershell-module-for-vmware-cloud-foundation-reporting/release-notes/?h=tar#v250 for the notice in the Release History.
